db_tasks.py

```diff
@@ -47,22 +69,52 @@ def databases_are_equal(dburl_a, dburl_b):
         return not m.statements
 
 
-def do_schema_dump(dburl, outfile):
-    COMMAND = 'pg_dump --no-owner --no-privileges --schema-only --column-inserts -f {} {}'
-
-    command = COMMAND.format(outfile, dburl)
-
-    print('MAKING DUMP OF SCHEMA: '.format(command))
-    subprocess.check_output(command, shell=True)
-    print('DUMP COMPLETE')
-
-
-def do_migration(REAL_DB_URL):
+def do_schema_dump(outfile, cld_url, export_url, username, password, service_binding):
+    ctx = ssl.create_default_context()
+
+    print('GETTING TOKEN')
+    conn = httplib.HTTPSConnection(cld_url, 443, context=ctx)
+    conn.request("POST", "/oauth/token", urllib.urlencode({
+        "grant_type": "password",
+        "scope": ",".join(["openid", "cloud_controller.read"]),
+        "username": username,
+        "password": password,
+    }), {
+        "Authorization": "Basic %s" % base64.b64encode("%s:%s" % ("db-export-tool-api", "notasecret")),
```

Comment:
I'd say it isn't worth it. They are the client ID / secret for the UAA client. It's a a bit of a longstanding OAuth wart that a client secret is required for installed app flows (which is close to what we're doing) - when that secret doesn't give any access on it's own (needs to be used along with a refresh token to fetch a new access token).